packs/trendmicro-cloudone-conformity/dashboards/trendmicro-cloudone-conformity.json

```diff
@@ -0,0 +1,373 @@
+              }
+            ],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT bytecountestimate()/1e9 AS 'GB' from TMCloudOneConformityEvent SINCE 1 month ago"
+              }
+            ],
+            "thresholds": []
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 3,
+            "row": 1,
+            "height": 3,
+            "width": 2
+          },
+          "title": "Compliance Score",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT round(percentage(uniqueCount(id, ruleId, status), WHERE status = 'SUCCESS')) as 'Percent' from TMCloudOneConformityEvent WHERE eventId IS NULL SINCE 1 month ago LIMIT MAX "
+              }
+            ],
+            "thresholds": [
+              {
+                "alertSeverity": "WARNING",
+                "value": 80
+              },
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 50
+              }
+            ]
+          },
+          "linkedEntityGuids": null
+        },
+        {
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "layout": {
+            "column": 5,
+            "row": 1,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Compliance % by Category",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT round(filter(uniqueCount(id, ruleId) * 100, WHERE status = 'SUCCESS' AND categories LIKE '%security%')/ filter(uniqueCount(id, ruleId), WHERE categories LIKE '%security%')) as Security, round(filter(uniqueCount(id, ruleId) * 100, WHERE status = 'SUCCESS' AND categories LIKE '%reliability%')/ filter(uniqueCount(id, ruleId), WHERE categories LIKE '%reliability%')) as Reliability, round(filter(uniqueCount(id, ruleId) * 100, WHERE status = 'SUCCESS' AND categories LIKE '%operational-excellence%')/ filter(uniqueCount(id, ruleId), WHERE categories LIKE '%operational-excellence%')) as 'Operational Excellence', round(filter(uniqueCount(id, ruleId) * 100, WHERE status = 'SUCCESS' AND categories LIKE '%performance-efficiency%')/ filter(uniqueCount(id, ruleId), WHERE categories LIKE '%performance-efficiency%')) as 'Performance Efficiency', round(filter(uniqueCount(id, ruleId) * 100, WHERE status = 'SUCCESS' AND categories LIKE '%cost-optimisation%')/ filter(uniqueCount(id, ruleId), WHERE categories LIKE '%cost-optimisation%')) as 'Cost Optimisation' from TMCloudOneConformityEvent  since 1 month ago limit max"
```

Comment:
This query seems to be rendering invalid as it's showing as plain text in GH...

Reply:
@aswanson-nr do we have any limits on the query field that may be causing this? I ran the query in Query Builder and it seem like a properly formatted NRQL query

Reply:
<img width="1859" alt="Screen Shot 2021-09-07 at 3 21 04 PM" src="https://user-images.githubusercontent.com/1395158/132412524-7fd906b9-2beb-4a6c-a4b7-15e5f0e09759.png">

Reply:
It's probably just because it's a very large string, if it was improper JSON syntax I would expect the rest of the file to have broken syntax coloring.

Reply:
Ok. thanks!